contrib/package.mill

```diff
@@ -126,18 +126,20 @@ object `package` extends RootModule {
 
     // Worker for Scoverage 2.0
     object worker2 extends build.MillPublishScalaModule {
-      def compileModuleDeps = Seq(build.main.api)
-      def moduleDeps = Seq(scoverage.api)
-      def testDepPaths = Task { Seq(compile().classes) }
+      def compileModuleDeps = Seq(
+        build.main.api,
+        scoverage.api
+      )
       def compileIvyDeps = Task {
-        Agg(
+        super.mandatoryIvyDeps() ++ Agg(
           // compile-time only, need to provide the correct scoverage version at runtime
           build.Deps.scalacScoverage2Plugin,
           build.Deps.scalacScoverage2Reporter,
           build.Deps.scalacScoverage2Domain,
           build.Deps.scalacScoverage2Serializer
         )
       }
+      def mandatoryIvyDeps = Agg.empty[Dep]
```

Comment:
What's going on with these changes? Did the semantics of the underlying resolution change in some way that we need to change this downstream code?

Reply:
The goal here is to also make the `mandatoryIvyDeps` (`scala-library` basically) compile-time only too. But that's just it.